Custom buttons (#9)
pdf-tools/pdf-web-viewer-samples: vanilla-typescript-examples/custom-buttons/README.md

# Custom buttons

> This example demonstrates how to create custom buttons inside PDF Web Viewer which can be used to execute code on click. Custom buttons can be used to add functionality to the PDF Web Viewer.

Reviewer: ```suggestion
This example shows how you can add your own custom buttons to the PDF Web Viewer toolbars. Custom buttons can be used to add functionality to the PDF Web Viewer.
```
